modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -126,17 +123,17 @@ lcb_install_callback3(instance, LCB_CALLBACK_GET, (lcb_RESPCALLBACK)get_callback
 
 lcb_STATUS rc;
 lcb_CMDGET *cmd;
-const char *collection = NULL, *scope = NULL;
-size_t collection_len = 0, scope_len = 0;
+const char *scope = "tenant_agent_00", *collection = "users";
 const char *key = "my-document";
 rc = lcb_cmdget_create(&cmd);
-rc = lcb_cmdget_collection(cmd, scope, scope_len, collection, collection_len);
+rc = lcb_cmdget_collection(cmd, scope, strlen(scope), collection, strlen(collection));
 rc = lcb_cmdget_key(cmd, key, strlen(key));
 rc = lcb_get(instance, NULL, cmd);
 rc = lcb_cmdget_destroy(cmd);
 rc = lcb_wait(instance);
 ----
 
+This code shows how you would use a named collection and scope. A named or default collection will provide the same functionality as bucket-level operations did in previous versions of Couchbase Server.
```

Comment:
Might be good to xref this to something from Couchbase Server's docs on collections?  This would be a good way for someone to understand "bucket-level operations" if needed.  Again, will defer to @RichardSmedley and team.

Reply:
:+1: